jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "allowSyntheticDefaultImports": true
+    },
+    "exclude": [
+        "node_modules"
+    ]
+}
```

Comment:
you can auto insert blank final lines with a setting in vscode 

https://github.com/Microsoft/vscode/issues/12076#issuecomment-272915821